Library/TestHelpers/XCTestCase+AppEnvironment.swift

```diff
@@ -71,6 +72,7 @@ extension XCTestCase {
         pushRegistrationType: pushRegistrationType,
         remoteConfigClient: remoteConfigClient,
         scheduler: scheduler,
+        stripeIntentService: stripeIntentService,
```

Comment:
Interesting idea to move this to `AppEnvironment`! Will that be sustainable when we have a lot of these services?

Reply:
Ohh that's a good point. _Probably not_ seeing as how AppEnvironment is already pretty large. That could be a good argument for just injecting this class into the ViewModel's that need it.

Reply:
Yeah, I'm OK with just injecting it where we need it. Unless we're starting to see issues with the tests for the VMs that use the `stripeIntentService` I'm not sure we need this level of DI